promql/promqltest/test.go

```diff
@@ -841,6 +865,8 @@ func (ev *evalCmd) checkAnnotations(expr string, annos annotations.Annotations)
 			warnings = append(warnings, err.Error())
 		case errors.Is(err, annotations.PromQLInfo):
 			infos = append(infos, err.Error())
+		default:
+			return fmt.Errorf("actual annotation must be either info or warn")
```

Comment:
Could you please include the unexpected error in this message? Otherwise it'd be very difficult to understand what went wrong here.

Reply:
Done!